vector/src/main/java/im/vector/fragments/VectorSettingsPreferencesFragment.java

```diff
@@ -1715,47 +1690,15 @@ private void refreshEmailsList() {
             // add new emails list
             mDisplayedEmails = newEmailsList;
 
-            int index = 0;
-            final Preference addEmailBtn = mUserSettingsCategory.findPreference(ADD_EMAIL_PREFERENCE_KEY);
-
-            // reported by GA
-            if (null == addEmailBtn) {
-                return;
-            }
-
-            int order = addEmailBtn.getOrder();
-
             for (final ThirdPartyIdentifier email3PID : currentEmail3PID) {
```

Comment:
It may worth to log a warning when there are more than one email in the account.
The case is not expected but it may be annoying to debug if it happens.

Reply:
I made a test and this case can not take place. With this evolution, in the project dinsic, the user can only fill one email (that of his registration). He can not add a second one, modify it or delete it from his account.
I do not know what are the permitted behaviors in the web version of Tchap?